models/public/open-closed-eye-0001/description/open-closed-eye-0001.md

```diff
@@ -0,0 +1,34 @@
+# Model name: action-recognition-0001-decoder
+
+## Use Case and High-Level Description
+
+Fully convolutional network for recognition of eye state ('open', 'closed').
+
+## Example
+
+![](./1.png)
+![](./2.png)
+![](./3.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Source framework                | PyTorch*                                  |
+| GFlops                          | 0.001                                     |
+| MParams                         | 0.01125                                   |
+
+
+## Performance
+
+## Inputs
```

Comment:
I think you should also adress @eizamaliev's comment https://github.com/opencv/open_model_zoo/pull/983#discussion_r396363590

In addition, please, leave some additional information if possible. Like datasets used to train or clarification of the use case and the metric values the net reaches like accuracy or whatever represent the quality best.